dasharo/project_config.yaml

```diff
@@ -0,0 +1,27 @@
+# Dasharo project configuration data
+
+client:
+        name: Immunefi
+
+framework:
+        name: dasharo
+        version: _
+        payload: seabios
```

Comment:
@miczyg1 we can say we use seabios for now?

Reply:
We build without payload now, but SeaBIOS is the target payload. I wonder how to treat it... Because in the next release SeaBIOS should boot already.